tsl/src/nodes/decompress_chunk/planner.c

```diff
@@ -382,7 +382,9 @@ decompress_chunk_plan_create(PlannerInfo *root, RelOptInfo *rel, CustomPath *pat
 	}
 	else if (IsA(compressed_path, BitmapHeapPath))
 	{
-		// TODO we should remove quals that are redunant with the Bitmap scan
+		// To increase performance, we should remove quals that are redunant with the Bitmap scan
```

Comment:
It locks bad and sloppy to keep commented code, so it should be removed altogether. I guess this should be put in an issue (including example code, if this is what this is) if there's no issue already.